### PR TITLE
fixed to_string

### PR DESCRIPTION
**Pull request by eford**
Compile error on g++ (GCC) 4.8.3, fixed here

### CODE REVIEW

File: Kepler-RoboVetter-DR25.cpp [hunk 1]
```diff
@@ -396,7 +396,8 @@ int main (int argc, char* argv[])
 
       // If it is the first iteration, save the major and minor flags.
       if(m==0)
-        data[n].disp_string = disp_pcfp + " " + to_string(data[n].not_transit_like) + " " + to_string(data[n].stellar_eclipse) + " " + to_string(data[n].centroid_offset) + " " + to_string(data[n].ephemeris_match) + " " + data[n].comments;
+        // data[n].disp_string = disp_pcfp + " " + to_string(data[n].not_transit_like) + " " + to_string(data[n].stellar_eclipse) + " " + to_string(data[n].centroid_offset) + " " + to_string(data[n].ephemeris_match) + " " + data[n].comments;
```
Comment (abdelkarim00) [inline Kepler-RoboVetter-DR25.cpp:399]: Remove the commented code.

File: Kepler-RoboVetter-DR25.cpp [hunk 2]
```diff
@@ -396,7 +396,8 @@ int main (int argc, char* argv[])
...
+        outfile << disp_pcfp << " " << data[n].not_transit_like << " " << data[n].stellar_eclipse << " " << data[n].centroid_offset << " " << data[n].ephemeris_match << " " << data[n].comments << "\n";
```
Comment (abdelkarim00) [inline Kepler-RoboVetter-DR25.cpp:400]: Why did you add a new line ('\n')?
 I think it doesn't exist in the original to_string.

**Comment by JeffLCoughlin** [comment]
Sorry Eric, I must have missed this pull request (the other year!).

The proposed edit (if I'm looking at the right thing) appears to write information to the output file via outfile <<, but what that line is supposed to do is save the original flags for later, until the whole line can be written out, containing the TCE, score (which requires the Monte Carlo to finish), and then that saved information.

I think the proper thing to-do would be to see if there is a replacement for to_string(), if that is causing the issue. Though it is supposed to be the C++11 standard way to do it.

If there's any interest left after this much time, what is the compile error?

**Comment by JeffLCoughlin** [comment]
Fixed by https://github.com/nasa/kepler-robovetter/commit/cd0ec1569ff491e32f72d7b8093bdd026cbe6d3f#diff-6d8adc65be61eecd30a6c9dc0de9e962
